### ISSUE TITLE
Social Icons: Centred icons no longer centred in the editor

### ISSUE THREAD

**Issue by iamtakashi** [Automattic/wp-calypso#41903]
This seems to be a WP.com specific issue. They correctly appear centred in my local install, but they are not centred in WP.com with the same theme. And this is an issue only in the editor as they correctly appear on the front of the site.

 **WP.com (simple site)**
<img width="1392" alt="Screenshot 2020-05-07 at 00 21 06" src="https://user-images.githubusercontent.com/908665/81238187-4e35d180-8ff9-11ea-8636-6898d390c739.png">

**Local install with the latest Gutenberg plugin**
<img width="1392" alt="Screenshot 2020-05-07 at 00 21 12" src="https://user-images.githubusercontent.com/908665/81238219-60177480-8ff9-11ea-8c95-8e9308f5c942.png">

Tested on Mac Chrome, theme: Hever

cc @ianstewart @alaczek

**Comment by Addison-Stavlo** [comment]
I just noticed this issue on my local gutenberg (last updated a few days ago).  After updating to current the issue is gone.

The problem was there was a `block-editor-block-list__block` between the `wp-block` and the `wp-block-social-links` which was messing up [the rule here](https://github.com/WordPress/gutenberg/blob/54c1b30cfe36e34edba521017c65036a661aac48/packages/block-library/src/social-links/editor.scss#L29-L31)

This seems to have been fixed recently.
